treeherder/webapp/api/utils.py

```diff
@@ -45,10 +45,14 @@ def get_end_of_day(date):
 
 def get_repository(param):
     """Returns repository id's by name"""
-    if param == 'trunk':
-        param = settings.TRUNK_REPO_NAMES
+    queryset = Repository.objects.values_list('id', flat=True)
 
+    if param == 'all':
+        return queryset.filter(active_status='active')
+
+    if param in settings.REPO_GROUPS:
```

Comment:
I think it might be clearer to not re-use the same variable here. Perhaps the method argument could be named `name` (or else `name_or_alias`), and then this becomes:
```py
if name in settings.REPO_GROUPSL:
    param = settings.REPO_GROUPS[name]
else:
    param = [name]
```